harc_game_web/apps/users/models.py

```diff
@@ -47,8 +47,8 @@ def score(self):
         # import is here, otherwise we have a cyclic import
         from apps.tasks.models import Task, DocumentedTask, TaskApproval
 
-        score = Task.objects.filter(documented_tasks__user=self.user).\
-                             filter(documented_tasks__taskapproval__is_accepted=True).\
+        score = Task.objects.filter(documentedtasks__user=self.user).\
```

Comment:
To w związku z tym usunęłaś related_name?

Reply:
też